app/javascript/components/subject/LastDateExposure.js

```diff
@@ -204,14 +204,41 @@ class LastDateExposure extends React.Component {
             </Row>
             <Row className="pt-2">
               <Col>
-                <Form.Check
-                  size="lg"
-                  label="CONTINUOUS EXPOSURE"
-                  type="switch"
-                  id="continuous_exposure"
-                  checked={this.state.continuous_exposure === true || false}
-                  onChange={() => this.toggleContinuousMonitoringModal()}
-                />
+                {!this.props.patient.monitoring && (
+                  <OverlayTrigger
+                    key="tooltip-ot-ce"
+                    placement="left"
+                    overlay={
+                      <Tooltip id="tooltip-ce">
+                        Continuous Exposure cannot be toggled for records on the Closed line list. If this monitoree requires monitoring due to a Continuous
+                        Exposure, you may update this field after changing Monitoring Status to &quot;Actively Monitoring&quot;
+                      </Tooltip>
+                    }>
+                    <span className="d-inline-block">
+                      <Form.Check
```

Comment:
Can we avoid the duplication here since the only difference in the case where it's closed is the `OverlayTrigger`? Something like moving the Form.Check code into a separate function.

Reply:
I considered it, but I think it's small enough (and only used twice in this file) that a dedicated function won't really save much.

Reply:
I'm thinking it will shorten the file and if we ever have to change the toggle again we only have to do it in one place vs. two.

Reply:
Made this change